collector/softnet_linux.go

```diff
@@ -61,10 +66,35 @@ func NewSoftnetCollector(logger log.Logger) (Collector, error) {
 			[]string{"cpu"}, nil,
 		),
 		timeSqueezed: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, softnetSubsystem, "times_squeezed_total"),
+			prometheus.BuildFQName(namespace, softnetSubsystem, "time_squeezed_total"),
```

Comment:
This is not a typo. Prometheus uses plural where appropriate.

This would also constitute a breaking change. Please leave this name as-is.

Reply:
done